tasks/s2i-nodejs.yaml

```
@@ -87,7 +89,9 @@ spec:
   - image: $(params.BUILDER_IMAGE)
     name: digest-to-results
     resources: {}
-    script: cat $(workspaces.source.path)/image-digest | tee /tekton/results/IMAGE_DIGEST
+    script: |
+      cat $(workspaces.source.path)/image-digest | tee $(results.IMAGE_DIGEST.path)
+      echo $(params.IMAGE) | tee $(results.IMAGE_URL.path)
```

```suggestion
      cat "$(workspaces.source.path)"/image-digest | tee $(results.IMAGE_DIGEST.path)
      echo "$(params.IMAGE)" | tee $(results.IMAGE_URL.path)
```
We should quote these as well.